rcsb/utils/dictionary/DictMethodCommonUtils.py

```diff
@@ -4098,20 +4115,20 @@ def getMaQaMetricType(self, dataContainer):
         maQaMetricLocalTypeD = {}
         maQaMetricGlobalTypeD = {}
 
-        compModelScoreTypeEnumD = {"zscore": "MA_QA_METRIC_LOCAL_TYPE_ZSCORE",
-                                   "energy": "MA_QA_METRIC_LOCAL_TYPE_ENERGY",
-                                   "distance": "MA_QA_METRIC_LOCAL_TYPE_DISTANCE",
-                                   "normalized score": "MA_QA_METRIC_LOCAL_TYPE_NORMALIZED_SCORE",
-                                   "pLDDT": "MA_QA_METRIC_LOCAL_TYPE_pLDDT",
-                                   "pLDDT in [0,1]": "MA_QA_METRIC_LOCAL_TYPE_pLDDT_[0,1]",
-                                   "pLDDT all-atom": "MA_QA_METRIC_LOCAL_TYPE_pLDDT_ALL-ATOM",
-                                   "pLDDT all-atom in [0,1]": "MA_QA_METRIC_LOCAL_TYPE_pLDDT_ALL-ATOM_[0,1]",
-                                   "PAE": "MA_QA_METRIC_LOCAL_TYPE_PAE",
-                                   "pTM": "MA_QA_METRIC_LOCAL_TYPE_pTM",
-                                   "ipTM": "MA_QA_METRIC_LOCAL_TYPE_ipTM",
-                                   "contact probability": "MA_QA_METRIC_LOCAL_TYPE_CONTACT_PROBABILITY",
-                                   "other": "MA_QA_METRIC_LOCAL_TYPE_OTHER"
-
+        compModelScoreTypeEnumD = {
+            "zscore": "MA_QA_METRIC_LOCAL_TYPE_ZSCORE",
+            "energy": "MA_QA_METRIC_LOCAL_TYPE_ENERGY",
+            "distance": "MA_QA_METRIC_LOCAL_TYPE_DISTANCE",
+            "normalized score": "MA_QA_METRIC_LOCAL_TYPE_NORMALIZED_SCORE",
+            "pLDDT": "MA_QA_METRIC_LOCAL_TYPE_pLDDT",
+            "pLDDT in [0,1]": "MA_QA_METRIC_LOCAL_TYPE_pLDDT_[0,1]",
+            "pLDDT all-atom": "MA_QA_METRIC_LOCAL_TYPE_pLDDT_ALL-ATOM",
+            "pLDDT all-atom in [0,1]": "MA_QA_METRIC_LOCAL_TYPE_pLDDT_ALL-ATOM_[0,1]",
+            "PAE": "MA_QA_METRIC_LOCAL_TYPE_PAE",
+            "pTM": "MA_QA_METRIC_LOCAL_TYPE_pTM",
+            "ipTM": "MA_QA_METRIC_LOCAL_TYPE_ipTM",
+            "contact probability": "MA_QA_METRIC_LOCAL_TYPE_CONTACT_PROBABILITY",
+            "other": "MA_QA_METRIC_LOCAL_TYPE_OTHER"
```

Comment:
I don't understand how this is missing in `staging`. Was this a merge issue?

Reply:
This was just a reformatting thing to appease pylint. When you define a multiline dictionary or tuple or list, the opening bracket should be on its own line, then all the items are listed with one indent inward, then the closing bracket must line up with the column that the variable was defined at. 

E.g.: 
```python
# Correct
myList = [
    "item1",
    "item2",
    ...
]

# Incorrect
myList = ["item1",    # <--item 1 should be returned to next line
    "item2",
    ...
    ]   # <-- should line up with myList

```

Reply:
Ok. I thought I fixed this pylint issue when I merged dev-comp-model with staging. Perhaps not.